packages/config/config/devices/0x0267/10002081-13x.json

```diff
@@ -0,0 +1,259 @@
+			"defaultValue": 39015,
+			"writeOnly": true,
+			"options": [
+				{
+					"label": "Parameters, Groups, and Z-Wave status are restored to default.",
+					"value": 39015
+				},
+				{
+					"label": "Parameters, except Lock long press are restored to default.",
+					"value": 17170
+				}
+			]
+		},
+		{
+			"#": "19",
+			"label": "Press Action",
+			"description": "Unlock/lock the direct control of load.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Pressing t <2sec switches between the last power on and off (depending on the status of the associated control) by sending a Basic Set to 0x00 or 0xFF to the associated nodes.",
+					"value": 0
+				},
+				{
+					"label": "Pressing t <2sec always activates those associated with the last poweron by sending aBasic Set to 0xFF.",
+					"value": 1
+				},
+				{
+					"label": "Pressing t <2sec always deactivates associates by sending a Basic Set to0x00.",
+					"value": 2
+				},
+				{
+					"label": "Pressing t <2sec on the left side sends a Basic Set to 0x00 and the DOWN 5s LED lights up. By pressing t <2sec on the right side a Basic Set is sent to 0xFF and the UP 5s led is lit.",
+					"value": 3
+				},
+				{
+					"label": "Pressing 0sec <t <2sec always activates the associated nodes by sending a BasicSet to 0xFF. Pressing 2sec <t <10sec always turns off the associated nodes by sending a Basic Set to 0x00. * Does not send Node Info nor does the central LED flash.",
+					"value": 4
+				},
+				{
+					"label": "Pressing t <2sec on the left side sends a Basic Set with VALUE OFF or 0x00 (depending on whether an OFF VALUE has been set). * If VALUEOFF is set, the DOWN LED lights up for 5sec. Otherwise the UP and DOWN LEDs will be activated briefly. Pressing t <2sec on the right side sends a Basic Set to VALUE ON or 0x00 (depending on whether an ON VALUE has been set). * If VALUE ON is set, the UP LED is lit for 5 seconds. Otherwise the UP and DOWN LEDs will be activated briefly.",
+					"value": 6
+				},
+				{
+					"label": "The associated nodes are controlled as described in the USE table KEYBOARD AND ANSWERS.",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "20",
+			"label": "Identify",
+			"valueSize": 1,
+			"minValue": 255,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"writeOnly": true,
+			"options": [
+				{
+					"label": "The central LED blinks for 5 seconds in order to identify the device.",
+					"value": 255
+				}
+			]
+		},
+		{
+			"#": "21",
+			"label": "Device State",
+			"valueSize": 2,
+			"minValue": 176,
+			"maxValue": 177,
+			"defaultValue": 176,
+			"readOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "0x00 Load is OFF. 0x01 Load is ON.",
+					"value": 176
+				},
+				{
+					"label": "0x00 Load is OFF. 0x01 Load is ON.",
+					"value": 177
+				}
+			]
```

Comment:
I don't understand this. Two options with the same label that seemingly have nothing to do with the values.